bootstrap/eks/config/manager/manager.yaml

```diff
@@ -25,5 +25,18 @@ spec:
         name: manager
       terminationGracePeriodSeconds: 10
       tolerations:
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/control-plane
         - effect: NoSchedule
           key: node-role.kubernetes.io/master
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Could we change this to `preferredDuringSchedulingIgnoredDuringExecution` instead?